core/src/main/java/org/elasticsearch/ingest/core/CompoundProcessor.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
+//TODO(simonw): can all these classes go into org.elasticsearch.ingest?
```

Comment:
the only way I see is to make ES depend on it an make it a sep jar just luke test-framework